www/FirebaseAuthentication.js

```diff
@@ -95,5 +95,54 @@ module.exports = {
         return new Promise(function(resolve, reject) {
             exec(resolve, reject, PLUGIN_NAME, "setLanguageCode", [null]);
         });
+    },
+     /**
+     * Updates the current user's profile data.
+     * 
+     * Passing a `null` value will delete the current attribute's value, but not
+     * passing a property won't change the current attribute's value:
+     *
+     * @example
```

Comment:
Lets remove `@example` block from jsdocs for now - in future API docs will be generated from source files, but at present it's simplier to have examples in README only.

Reply:
Removed js doc